src/daemon/uploader.cpp

```diff
@@ -35,14 +36,35 @@ void Uploader::triggerSync()
         return;
     }
 
-    if (this->m_webDavCommandQueue->queue().length() > 0)
+    // detect if sync for "localPath" is in the queue already
+    if (m_syncingPaths.contains(localPath)){
+        qWarning() << "Syncing already:" << localPath;
         return;
+    }
+
+    if (this->m_webDavCommandQueue->queue().length() > 64) {
```

Comment:
What's the reason for this check?

Reply:
as there is another check `m_syncingPaths.contains(localPath)`, this check has no sense now... removing it...